f5/bigip/mixins.py

```diff
@@ -108,3 +108,52 @@ def __setattr__(self, key, value):
                     [self.__dict__.pop(n, '') for n in new_set]
         # Now set the attribute
         super(ExclusiveAttributesMixin, self).__setattr__(key, value)
+
+
+class UnnamedResourceMixin(object):
+    '''This makes a resource object work if there is no name.
+
+    These objects do not support create or delete and are often found
+    as Resources that are under an organizing collection.  For example
+    the `mgmt/tm/sys/global-settings` is one of these and has a kind of
+    `tm:sys:global-settings:global-settingsstate` and the URI does not
+    match the kind.
+    '''
```

Comment:
There is no unique component/element/part of this resources URI-path.   We still say it's a Resource, because it's obviously NOT a Collection, _but_ it obviously can't support `create`, or `delete` since those operations are intrinsically tied to the unique parts of the URI (maybe we should add a check to delete! That says, if no `self._meta_data['uri']` then raise!).

If I Understand Correctly (IIUC) then I think we should also remove support for `load`, since it adds nothing that `refresh` doesn't supply...   right?!!?   Do we allow `load` on e.g. OrganizingCollections?